sim_pipeline/halos/get_halos.ipynb

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    halos_lens = HalosLens(halos_list=halos, sky_area=0.00001)
</pre><p>might be worth making a cosmology instance such that it is well reproducible and no UserWarning is raised in the output line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sibirrer/sim-pipeline/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>